contracts/multisigs/GnosisSafeMultisig.sol

```diff
@@ -14,11 +14,18 @@ interface IGnosisSafeProxyFactory {
     ) external returns (address proxy);
 }
 
+/// @dev Provided incorrect data length.
+/// @param expected Expected minimum data length.
+/// @param provided Provided data length.
+error IncorrectDataLength(uint256 expected, uint256 provided);
+
 /// @title Gnosis Safe - Smart contract for Gnosis Safe multisig implementation of a generic multisig interface
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
 contract GnosisSafeMultisig {
     // Selector of the Gnosis Safe setup function
-    bytes4 internal constant _GNOSIS_SAFE_SETUP_SELECTOR = 0xb63e800d;
+    bytes4 public constant GNOSIS_SAFE_SETUP_SELECTOR = 0xb63e800d;
+    // Default data size to be parsed and passed to the Gnosis Safe Factory without payload
+    uint256 public constant DEFAULT_DATA_LENGTH = 144;
```

Comment:
Adding the default data length when the passed data length is different from the zero